src/components/SpeciesDeliverableTable/TableCellRenderer.tsx

```diff
@@ -41,7 +45,16 @@ export default function SpeciesDeliverableCellRenderer(props: RendererProps<Tabl
 
   const createLinkToAcceleratorSpecies = (iValue: React.ReactNode | unknown[]) => {
     return (
-      <Link to={`${APP_PATHS.ACCELERATOR_SPECIES.replace(':speciesId', row.speciesId.toString())}`}>
+      <Link
+        onClick={() => {
+          setCurrentParticipantProjectSpecies(row.id);
+          navigate(
```

Comment:
Can you add this to https://github.com/terraware/terraware-web/blob/main/src/hooks/useNavigateTo.ts#L7?